src/Components/CollectionsHubRails/ArtistSeriesRail/__tests__/ArtistSeriesEntity.test.tsx

```diff
@@ -13,34 +13,86 @@ describe("ArtistSeriesEntity", () => {
     }
   })
 
-  it("Renders expected fields", () => {
+  it("showing the correct text, price guidance, amount of hits and image", () => {
     const component = mount(<ArtistSeriesEntity {...props} />)
-
-    expect(component.text()).not.toMatch("Jasper Johns:")
-    expect(component.text()).toMatch("Flag")
+    expect(component.text()).toMatch("Flags unique collections")
     expect(component.text()).toMatch("From $1,000")
     expect(component.find(ArtworkImage).length).toBe(3)
     const artworkImage = component
       .find(ArtworkImage)
       .at(0)
       .getElement().props
-
     expect(artworkImage.src).toBe(
       "https://d32dm0rphc51dk.cloudfront.net/4izTOpDv-ew-g1RFXeREcQ/small.jpg"
     )
-    expect(artworkImage.alt).toBe("Jasper Johns, Flag")
+  })
+
+  it("NOT rendering any artist name or the title of the artwork ", () => {
+    const component = mount(<ArtistSeriesEntity {...props} />)
```

Comment:
I'm still unsure why we are verifying that these aren't appearing. Is there value in not showing these? If so, then the test name should tell us _why_ it's import - like `it("doesn't render artist name or title because they XXXXX")`. 

If the test is verifying that we didn't render these fields to be certain that we didn't write code to render them..those types of assertions are typically avoided in tests. Writing tests that verify "we didn't do XXX" comprehensively would take a really long time, and it doesn't give enough value to justify them.

If you want, we can talk about this some more in person!

Reply:
Because there are two 'title' in the query, one is for whole group of the collection and one is for single artwork in the group. ( You can find at "src/Apps/__tests__/Fixtures/Collections.ts", one is under the "members", and one is under "hits" . ) I want to make sure the component is rendering the title from the whole group of the collection, not the title of the single artwork. 
The assertion seems it is also misleading people. So, for this one, I will remove this, you are right. It seems it is unnecessary since the test before this one just check the title and text.